### DISCUSSION TITLE
0Auth - Redirect URI make HTTPS instead of HTTP ?

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#9670] [Archive]
Is it possible to set the redirect url to https instead of http ?

**Comment by rijkvanzanten** [comment]
I'm assuming you're talking about the default redirect url that's appended when you're using OAuth on the login page. If that's the case, then it's a matter of updating your PUBLIC_URL environment variable 👍🏻

**Reply by NilsBaumgartner1994** [reply]
Yes, thanks
